views/home.erb

```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>SCRABBLE!!!</h1>
+    <p>
+      Go To the <strong>Score</strong> page to score one word or the <strong>Score Multiple Words</strong> page to score multiple words.
```

Comment:
This would be a nice place to include a link to these pages as well (even though I know they're also on the nav bar)